boards/arm/mps2_an521/CMakeLists.txt

```diff
@@ -7,3 +7,24 @@
 zephyr_library()
 zephyr_library_sources(pinmux.c)
 zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
+
+if(CONFIG_SOC_MPS2_AN521_CPU1)
+# Building a firmware image for CPU1: this requires a binary
+# for CPU0, which will boot the device and wake up CPU1.
```

Comment:
If user builds a dedicated sample for CPU0 himself, then how can he avoid building the `empty_cpu0` sample ?

Reply:
Looks, like we cannot get around without a new Kconfig option indicating dual core builds, e.g. config BOARD_MPS2_AN521_BUILD_FOR_STANDALONE_CPU1